### ISSUE TITLE
Add old symCIF data names

### ISSUE THREAD

**Issue by jamesrhester** [COMCIFS/MultiBlock_Dictionary#10]
A few symCIF data names only made sense when more than one space group was referenced. This requires using some identifiers from the multi-block dictionary to identify those space groups.

**Comment by rowlesmr** [comment]
Is symCIF available in DDLm format?

**Comment by jamesrhester** [comment]
> Is symCIF available in DDLm format?

No, it has almost entirely been merged into the DDLm dictionaries.
